vnext/Scripts/Microsoft.ReactNative.targets

```diff
@@ -3,27 +3,17 @@
   <PropertyGroup>
     <Native-Platform Condition="'$(Platform)' == 'Win32'">x86</Native-Platform>
     <Native-Platform Condition="'$(Platform)' != 'Win32'">$(Platform)</Native-Platform>
-    <_rnwDebugFolder>$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\debug\</_rnwDebugFolder>
-    <_rnwReleaseFolder>$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\release\</_rnwReleaseFolder>
+    <_rnwFolder>$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\</_rnwFolder>
```

Comment:
(nit) Following MSBuild conventions, call use Pascal case, and use the "Dir" prefix?
(i.e. `_RnwDir`).

Reply:
As this pattern was already there. I'll address this in the main branch, and not do this in the backport...

---
In reply to: [516984116](https://github.com/microsoft/react-native-windows/pull/6417#discussion_r516984116) [](ancestors = 516984116)